### ISSUE TITLE
Disabling "remove widget" for a set of widgets

### ISSUE THREAD

**Issue by hmny** [angular-dashboard-framework/angular-dashboard-framework#193]
Is there any `widgets` attribute that can prevent a widget to get removed? So something like a `removeable: true/false` in mainController that generates a default set of widgets that always should be on the screen:
```javascript
rows:[
{
	columns: [{
		styleClass: "col-md-6",
		widgets: [{
			type: "baseWidget",
			removeable: false
		}]
	}
}]
```

**Comment by mnj11** [comment]
Same question here, is there any such flag ?

**Comment by hmny** [comment]
@mnj11 I'm assuming there isn't any since I didn't get any response. Adding a "default" widget was the closest solution I came up with

**Comment by mnj11** [comment]
@hmny  yeah, I guess so..in this case do you know if there is any flag that allows user to enable/disable a widget on the fly by say setting something like enable: true or false ?

**Comment by hmny** [comment]
Unfotunately I am not awaire of that functionality
